schema/reportdb/common/tables/ChannelContentSource.sql

```diff
@@ -0,0 +1,25 @@
+--
+-- Copyright (c) 2022 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+
+CREATE TABLE ChannelContentSource
+(
+    mgm_id                    NUMERIC NOT NULL,
+    channel_id                NUMERIC NOT NULL,
+    content_source_id         NUMERIC NOT NULL,
+    label                     VARCHAR(128),
+    url                       VARCHAR(100),
+    type                      VARCHAR(64),
+    metadata_signed           BOOLEAN,
+    synced_date               TIMESTAMPTZ DEFAULT (current_timestamp)
+);
```

Comment:
This looks like a join table, but it is not. I think content_source_id exists only to make the primary key unique.

I would suggest to rename this table to "Repository" and maybe the column content_source_id to repository_id.

Reply:
It was meant to be a join table actually but it was duplicating the repository data. So I introduced a new Repository table and renamed this one to ChannelRepository